plugin/pkg/scheduler/algorithm/listers.go

```diff
@@ -91,3 +91,39 @@ func (f FakeServiceLister) GetPodServices(pod *api.Pod) (services []api.Service,
 
 	return
 }
+
+// ControllerLister interface represents anything that can produce a list of ReplicationController; the list is consumed by a scheduler.
+type ControllerLister interface {
+	// Lists all the replication controllers
+	List() ([]api.ReplicationController, error)
+	// Gets the services for the given pod
+	GetPodControllers(*api.Pod) ([]api.ReplicationController, error)
```

Comment:
I used the same name that's used in https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/client/cache/listers.go#L199.